lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -500,6 +510,17 @@ class Simulator {
   static get ALL_NODE_TIMINGS() {
     return ALL_SIMULATION_NODE_TIMINGS;
   }
+
+  /**
+   * We attempt to start nodes by their observed start time using the record priority as a tie breaker.
+   * When simulating, just because a low priority image started 5ms before a high priority image doesn't mean
+   * it would have happened like that when the network was slower.
+   * @param {Node} node
+   */
+  static _computeNodeStartPosition(node) {
+    if (node.type === 'cpu') return node.startTime;
+    return node.startTime + (PriorityStartTimePenalty[node.record.priority] * 1000 * 1000 || 0);
```

Comment:
wait, this seems different than just a tie breaker. It can actually change the order of nodes that don't have the same start time?

Reply:
"tiebreaker" if it were only the *exact* same nanosecond timestamp is useless. The lower the priority the wider the berth that's given to consider it a "tie" and defer to the higher priority request.

Requests started within the same second that all have their dependencies finished do not really need to be started in the exact same observed order that Chrome happened to request them.

Reply:
> "tiebreaker" if it were only the _exact_ same nanosecond timestamp is useless.

sure, but usually "tie breaker" is for exactly that sort of thing (e.g. sorting comparators). This is more of a time spreading heuristic :)

You didn't post much motivation for the exact numbers used in the PR, and it's not clear to me that reordering like that would work out correctly, but I guess the proof is the comparison errors :)

Reply:
> I guess the proof is the comparison errors

bingo :) haha 

and that's fair, so here's my caveat to anyone who comes back trying to find where they came from

Dear reader from the future, I do not have supreme confidence that the exact numbers used in the PR are optimal nor were they divined after some extensive research process. They "felt right" and improved accuracy so if you need to change them for similar reasons, feel free :)

Reply:
lmao i am future reader